py/picca/io.py

```diff
@@ -770,18 +578,18 @@ def read_from_mock_1d(filename,
     except IOError:
         log_file.write("error reading {}\n".format(filename))
 
-    for t, r, d, z, p, m, f in zip(thingid, ra, dec, z_qso, plate, mjd,
-                                   fiberid):
-        hdu = hdul["{}".format(t)]
-        log_file.write("file: {} hdus {} read  \n".format(filename, hdu))
-        lambda_ = hdu["wavelength"][:]
-        log_lambda = np.log10(lambda_)
+    for entry in catalog:
+        thing_id = entry['THING_ID']
+        hdu = hdul[f'{thing_id}']
```

Comment:
why not use the .format() ? I think it is a bit clearer than the f"  " notation

Reply:
I disagree. How to solve this conflict ? :)

Reply:
I'm not strong headed here. I guess we can leave this with the f notation, but maybe @Waelthus can break the tie

Reply:
I tend to prefer the f-string for variable names or very short operations and the format for longer operations, in this case I'd be on @julianbautista's side

Reply:
ok, let's keep it with the f format then